.github/build_real.sh

```diff
@@ -2,4 +2,4 @@
 set -e
 cp $CONFIG_FILE config/config.mk
 make
-pip install .
+pip install -e .
```

Comment:
I think so because it's necessary for codecov to work properly.

Reply:
I'm fairly certain that we don't need to do this (it's kinda silly to enforce all coverage testing to be installed in developer mode) but I haven't had time to figure out how to do it otherwise.